system-tests/e2e-suite/src/test/scala/com/hotels/styx/client/ExpiringConnectionSpec.scala

```diff
@@ -36,31 +41,38 @@ import scala.concurrent.duration._
 
 class ExpiringConnectionSpec extends FunSpec
   with StyxProxySpec
-  with StyxClientSupplier
   with Eventually {
 
   val mockServer = FakeHttpServer.HttpStartupConfig()
     .start()
     .stub(urlStartingWith("/app1"), aResponse
       .withStatus(200)
+      .withHeader("ExpiringConnectionSpec", "x")
       .withHeader(TRANSFER_ENCODING, CHUNKED)
       .withBody("I should be here!")
     )
 
-  override val styxConfig = StyxConfig()
+  val styxConfig = StyxConfig()
```

Comment:
You can remove the `styxConfig` value, provided that you'll mix in the `DefaultStyxConfig` trait like so:

```
class ExpiringConnectionSpec extends FunSpec
  with StyxProxySpec
  with DefaultStyxConfiguration
  with Eventually {
```